src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -361,6 +362,20 @@ export default class DissolutionFirm extends Mixins(DateMixin) {
     return emailList
   }
 
+  // FUTURE: pull this from shared auth composable (after vue3 upgrade)
+  get isSbcStaff (): boolean {
+    const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+    console.log(currentAccount)
+    if (!currentAccount) return false
+    try {
+      console.log(JSON.parse(currentAccount).accountType)
+      return JSON.parse(currentAccount).accountType === AccountTypes.SBC_STAFF
+    } catch (error) {
+      console.log('Error parsing current account =', error)
+      return false
+    }
+  }
```

Comment:
As in Edit UI, this already exists in App.vue.

Just add a new getter to state-getters.ts.